requirements/requirements-testing.txt

```diff
@@ -0,0 +1,5 @@
+pytest>=4.6.1,<4.7 # 4.6 is the last version to support python 2.7
+pytest-django==3.9.0
+pytest-cov==2.9.0 # coverage report
```

Comment:
If you won't use pytest but only django tests then these dependencies are not required.